test/object_storage/test_azure.py

```diff
@@ -103,3 +103,24 @@ def test_get_contents_to_fileobj_raises_error_on_invalid_byte_range(azure_module
             fileobj_to_store_to=BytesIO(),
             byte_range=(100, 10),
         )
+
+
+def test_conn_string_custom_host_port() -> None:
```

Comment:
Since there is a strange behavior on I pointed out above, I suggest to write tests without custom host and port as well.